adam/ontology/phase1_ontology.py

```diff
@@ -242,15 +256,114 @@ def subtype(sub: OntologyNode, _super: OntologyNode) -> None:
 subtype(DESTINATION, SEMANTIC_ROLE)
 
 # Hierarchical structure of objects
-_HEAD_SCHEMA = ObjectStructuralSchema(_HEAD)
 _TORSO_SCHEMA = ObjectStructuralSchema(_TORSO)
-_ARM_SCHEMA = ObjectStructuralSchema(_ARM)
+# _ARM_SCHEMA = ObjectStructuralSchema(_ARM)
 _LEG_SCHEMA = ObjectStructuralSchema(_LEG)
 _CHAIRBACK_SCHEMA = ObjectStructuralSchema(_CHAIR_BACK)
 _CHAIR_SEAT_SCHEMA = ObjectStructuralSchema(_CHAIR_SEAT)
 _TABLETOP_SCHEMA = ObjectStructuralSchema(_TABLETOP)
 _TAIL_SCHEMA = ObjectStructuralSchema(_TAIL)
 _WING_SCHEMA = ObjectStructuralSchema(_WING)
+_EYE_SCHEMA = ObjectStructuralSchema(_EYE)
+_MOUTH_SCHEMA = ObjectStructuralSchema(_MOUTH)
+_EAR_SCHEMA = ObjectStructuralSchema(_EAR)
+_NOSE_SCHEMA = ObjectStructuralSchema(_NOSE)
+_FINGER_SCHEMA = ObjectStructuralSchema(_FINGER)
+_PALM_SCHEMA = ObjectStructuralSchema(_PALM)
+_BONE_SCHEMA = ObjectStructuralSchema(_BONE)
+_WINDOW_SCHEMA = ObjectStructuralSchema(_WINDOW)
+_ROOF_SCHEMA = ObjectStructuralSchema(_ROOF)
+_WALL_SCHEMA = ObjectStructuralSchema(_WALL)
+_TIRE_SCHEMA = ObjectStructuralSchema(_TIRE)
+_FLATBED_SCHEMA = ObjectStructuralSchema(_FLATBED)
+
+# Structural Schema Below are Vocabulary Options
+_DOOR_SCHEMA = ObjectStructuralSchema(DOOR)
```

Comment:
If these are objects without any internal structure (currently), then pull this block up above the more detailed breakdowns and intermediate objects (like e.g. `_TAIL_SCHEMA`) and put a comment to that effect.